app/models/pipeline_run.rb

```diff
@@ -901,7 +901,7 @@ def check_and_log_long_run
     MetricUtil.put_metric_now("samples.running.run_time", run_time, tags, "gauge")
 
     if alert_sent.zero?
-      threshold = 8.hours
+      threshold = 10.hours
```

Comment:
it's always nice to add a comment for constant such as these as to how the value was determined. for example "queries last 1000 pipline runs, found 95% finished in less than 10 hours"

Reply:
👍